src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -19,6 +19,7 @@
 using Raven.Server.Documents.Includes;
 using Raven.Server.Documents.Indexes.Auto;
 using Raven.Server.Documents.Indexes.MapReduce.Auto;
+using Raven.Server.Documents.Indexes.MapReduce.Static;
```

Comment:
Also, `MapFields` is created with `OrdinalIgnoreCase`, and I'm pretty sure that field names are case sensitive